src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -33,24 +33,28 @@ export const coreVitals = (): void => {
 		name: string;
 		value: number;
 	};
+	
+	const nineDigitPrecision = (number: number) => {
+		// The math functions are to make sure the length of number is <= 9
```

Comment:
Does this length of `9` includes the decimal point?

Reply:
And why 9 specifically? Is this a requirement from Data Tech?

Reply:
We use 9 so it will fit in the body we send to Fastly. We tried different combinations (high numbers would fail) and found this worked and gave us more than enough overhead on the data granularity.

Reply:
Cool, when we look at this, we're likely to round to the nearest millisecond. I wonder if the sub-millisecond precision is brining a lot of value in your case?

https://developer.mozilla.org/en-US/docs/Web/API/DOMHighResTimeStamp

Also curious what happens if you have values beyond 999ms, because then you'd get 10 digits. Maybe the method could be renamed to `roundBeyondSixDecimal`.

Reply:
With core vitals it would be reported as 999, they send different formats for each metric.